starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,46 @@
+body{
+  font-family: Arial, Helvetica, sans-serif;
+}
+h1{
+  color: white;
+  background-image: url('/Users/Kristen/Desktop/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg');
+  background-repeat: no-repeat;
+  height: 500px;
+  width: 80%;
```

Comment:
no need to adjust width if you want background image to cover the size of the div unless you want to make the image smalller.